### DISCUSSION TITLE
Webscrapping dengan R, format web card

### DISCUSSION THREAD

**Discussion by yuliristiana** [teamalgoritma/community#825] [Q&A] [answered]
Halo,

Mau tanya cara/step/atau referensi untuk pengambilan data cara webscrapping menggunkan R. Dan untuk webnya sendiri berbentuk card bukan yang sudah dalam bentuk table.

info -> web https://alternativebrewing.com.au/

terima kasih..

**Comment by Yusraf002** [comment, marked as answer]
Dear @yuliristiana 

Untuk melakukan webscraping terhadap card dari link yang telah disertakan, Ibu dapat mengikuti panduan pada website berikut: https://www.zenrows.com/blog/web-scraping-r#r-web-crawling. Di sana, tersedia contoh dan langkah-langkah untuk melakukan web scraping menggunakan R dengan package `rvest`. Untuk informasi lebih lanjut, silakan baca dan akses melalui link tersebut.

Semoga informasi ini membantu. 
Terimakasih telah bertanya dan selamat melanjutkan aktivitasnya.

Best Regards,
Team Algoritma.

**Reply by yuliristiana** [reply]
Terima kasiih mas Yusraf atas jawabannya
